core/coreapi/name.go

```diff
@@ -89,9 +89,7 @@ func (api *NameAPI) Publish(ctx context.Context, p coreiface.Path, opts ...caopt
 	}, nil
 }
 
-// Resolve attempts to resolve the newest version of the specified name and
-// returns its path.
-func (api *NameAPI) Resolve(ctx context.Context, name string, opts ...caopts.NameResolveOption) (coreiface.Path, error) {
+func (api *NameAPI) Search(ctx context.Context, name string, opts ...caopts.NameResolveOption) (<-chan coreiface.IpnsResult, error) {
```

Comment:
Please document this function.

Reply:
There are docs for this on the interface (like all other coreapi functions). We may want to point people there, but I'd this in a separate PR

Reply:
Let's just do this now. Separate PRs for documentation tend to not happen.